.travis.yml

```diff
@@ -23,17 +25,17 @@ script:
 
 after_success:
   - codecov
+  - if [[ $BUILD_INSTALLER == 'yes' ]]; then pip install pynsist; ./script/makeinstaller.sh; fi
```

Comment:
Fair enough, been a while since I did much travis work. Yeah `pynsist` is better in the `before_install`, you're right.